### PR TITLE
Rename tests with conflicting names 

### PR DESCRIPTION
**Pull request by aasaru**
tests exist with same name and package in https://github.com/apache/james-project/tree/master/server/mailet/mailets/src/test/java/org/apache/james/transport/mailets

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]
Good catch!

Thanks

File: server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ToRepositoryIngtegrationTest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
 import io.restassured.specification.RequestSpecification;
 
-public class ToRepositoryTest {
+public class ToRepositoryIngtegrationTest {
```
Comment (Arsnael) [inline server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ToRepositoryIngtegrationTest.java:54]:
```suggestion
public class ToRepositoryIntegrationTest {
```

**Review by Arsnael — COMMENTED** [review]
Hi! Just a small typo spotted, but overall thanks for the fix :)

**Review by Arsnael — APPROVED** [review]

**Comment by aasaru** [comment]
> Hi! Just a small typo spotted, but overall thanks for the fix :)

Good catch, I didn't notice this myself! I approved the fix.

**Comment by Arsnael** [comment]
Hi @aasaru , your PR has been merged, thanks for your contribution :) 

Could you please help me closing your PR?

**Pull request closed without merging**